plugins/project-manager/README.md

```
@@ -1,11 +1,11 @@
 # project-manager
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
-[![Skills](https://img.shields.io/badge/Skills-3-blue.svg)]()
+[![Skills](https://img.shields.io/badge/Skills-4-blue.svg)]()
```

markdownlint-cli2 (0.22.0)

[warning] No empty links

(MD042, no-empty-links)